can_tools/scrapers/official/PA/philadelhpia_vaccine.py

```diff
@@ -23,23 +23,24 @@ class PhiladelphiaVaccine(TableauDashboard):
     viewPath = "COVIDVaccineDashboard/COVID_Vaccine"
     data_tableau_table = "Residents Percentage {dose_type}"
     variables = {
-        "Residents Receiving At Least 1 Dose* ": variables.INITIATING_VACCINATIONS_ALL,
-        "Fully Vaccinated Residents*": variables.FULLY_VACCINATED_ALL,
+        "New": variables.INITIATING_VACCINATIONS_ALL,
+        "Full": variables.FULLY_VACCINATED_ALL,
+        "Booster": variables.PEOPLE_VACCINATED_ADDITIONAL_DOSE,
     }
 
     def fetch(self) -> pd.DataFrame:
         # create a dict of the 2 dose type tables
         # which are titled "Residents Percentage First" and "... Full"
         return {
             dose_type: self.get_tableau_view(dose_type=dose_type)[
-                self.data_tableau_table.format(dose_type=table_name)
+                self.data_tableau_table.format(dose_type=dose_type)
             ]
-            for dose_type, table_name in {"First": "New", "Full": "Full"}.items()
+            for dose_type in ["New", "Full", "Booster"]
```

Comment:
They've seemed to have switched back from the changes in #402, so I've reverted the code here to match